src/api/item.ts

```diff
@@ -27,16 +32,26 @@ export async function getSubcategoryItems(
         next: { tags: ['items'] },
       }
     );
-    return await res.json();
+    const result = await res.json();
+    return {
+      error: result.error,
+      subcategoryItems: result.subcategoryItems,
+      message: result.message,
+    } as SubCategoryItemsResponse;
```

Comment:
We could avoid this `as` and below by converting error from unknown type to string like:
```
catch (e) {
    console.log('error', e);
    const eString = JSON.stringify(e);

    return {
      error: eString,
      subcategoryItem: null,
      message: eString,
    };
  } 
```

Reply:
without **as** it does not know what type comes

Reply:
It should know because we write return type in the start of the function Promise<SubCategoryItemsResponse>

In catch there is also could be a situation when error is not that error which was send from response. So in catch an error is always unknown.

Reply:
ok, I understand! Thanks :)